tests/all.sh

```diff
@@ -55,3 +50,28 @@ RUST_BACKTRACE=1 RUST_LOG=info cargo test -- --test-threads=1 --nocapture
 # Stop TPM server #
 ###################
 pkill tpm_server
+
+#############################
+# Install nightly toolchain #
+#############################
+rustup toolchain install nightly
+
+############################
+# Install legacy toolchain #
+############################
+rustup toolchain install 1.38.0
+
+####################
+# Verify doc build #
+####################
+cargo +nightly doc --features docs --verbose
+
+########################
+# Verify nightly build #
+########################
+cargo +nightly build
```

Comment:
Should we add a requirement on this crate that it has to build on nightly?

Reply:
Maybe a note on the doc landing page that the crate is expected to build successfully under the nightly compiler and every Rust compiler since 1.38.0